.github/workflows/ci.yml

```diff
@@ -130,6 +130,8 @@ jobs:
         working-directory: doc
       - name: Copy API
         run: cp -r API public/api
+      - name: Add CNAME
```

Comment:
I think we could add this file to the .vuepress/public folder and then it would automatically be copied.